spec.html

```diff
@@ -34788,12 +34822,13 @@ <h1>%TypedArray%.prototype.set ( _array_ [ , _offset_ ] )</h1>
             1. Let _targetByteOffset_ be _target_.[[ByteOffset]].
             1. Let _src_ be ? ToObject(_array_).
             1. Let _srcLength_ be ? LengthOfArrayLike(_src_).
+            1. If _targetOffset_ is +&infin;, throw a *RangeError* exception.
```

Comment:
To confirm my understanding, this is to properly define using infinity in the addition in the RangeError check below, which was previously undefined.

Reply:
Yup. And it has to happen at this point for side effect order reasons, even though it might be more natural earlier.